doc/api/cli.md

```diff
@@ -340,6 +358,16 @@ misformatted, but any errors are otherwise ignored.
 Note that neither the well known nor extra certificates are used when the `ca`
 options property is explicitly specified for a TLS or HTTPS client or server.
 
+### `SSL_CERT_DIR=dir`
+
+If `--use-openssl-ca` is enabled, this overrides and sets OpenSSL's directory
+containing trusted certificates.
+
+### `SSL_CERT_FILE=file`
```

Comment:
What is the potential risk of allowing the user to decide for themselves what CAs they trust? Should document the potential risk of having the certs compiled in... where its impossible for them to be changed if sec policy requires it.

Inherited, perhaps a note such as:

> Note: Be aware that unless the child environment is explicitly set, this evironment variable will be inherited by any child processes, and if they use OpenSSL, it may cause them to trust the same CAs as node.